apps/condo/domains/notification/transports/push.js

```diff
@@ -86,17 +105,17 @@ const mixResult = (container, result) => {
 }
 
 /**
- * Send notification using corresponding transports (depending on FireBase/Huawei and appId)
+ * Send notification using corresponding transports (depending on FireBase/Huawei/Apple and appId)
  * @param notification
  * @param data
  * @param user
  * @param remoteClient
  * @returns {Promise<[boolean, {error: string}]|(boolean|{})[]>}
  */
-async function send ({ notification, data, user, remoteClient } = {}) {
+async function send ({ notification, data, user, remoteClient } = {}, isVoIP = false) {
```

Comment:
It's would good to update jsdoc after the function signature updated

Reply:
I meant parameters :)

Reply:
fixed